libsplinter/src/circuit/template/mod.rs

```diff
@@ -16,39 +16,61 @@ mod error;
 mod rules;
 mod yaml_parser;
 
-use std::collections::HashMap;
-
 pub use error::CircuitTemplateError;
 
-use rules::CircuitCreateTemplate;
-use yaml_parser::CircuitTemplate;
+use rules::{RuleArgument, Rules};
```

Comment:
I'm not sure I understand the commit description (I think there may be a typo); could you reword it? This is the "Move CircuitCreateTemplate to module root file" commit

Reply:
Addressed